cypress/integration/cucumber/step_definitions/refapp-3.x/04-clinical-visit/clinical-visit.js

```diff
@@ -0,0 +1,177 @@
+import {Given, When} from 'cypress-cucumber-preprocessor/steps';
+
+let patient = null;
+
+beforeEach("Take a dump of the database and restore",() => {
+    //Run the script using npm
+    cy.exec("npm run resetDB")
+})
```

Comment:
We can remove this hook because I added it to the commands.js file. Now it runs for each test case. We do not need to add this hook for every test file.

Reply:
thanks for the review @pasindur99 
This is still failing but i am gonna fix it over the weekend